saagieapi/docker_credentials/docker_credentials.py

```diff
@@ -9,20 +10,24 @@ class DockerCredentials:
     def __init__(self, saagie_api):
         self.saagie_api = saagie_api
 
-    def list_for_project(self, project_id: str) -> Dict:
+    def list_for_project(self, project_id: str, pprint_result: Optional[bool] = True) -> Dict:
         """
         Get all saved docker credentials for a specific project
         Parameters
         ----------
         project_id : str
             ID of the project
+        pprint_result : str
+            Whether to pretty print the results in the console or not
         Returns
         -------
         dict
 
         """
         params = {"projectId": project_id}
-        return self.saagie_api.client.execute(gql(GQL_GET_ALL_DOCKER_CREDENTIALS), variable_values=params)
+        return self.saagie_api.client.execute(
+            query=gql(GQL_GET_ALL_DOCKER_CREDENTIALS), variable_values=params, pprint_result=pprint_result
+        )
 
     def get_info(self, project_id: str, credential_id: str) -> Dict:
```

Comment:
Add `pprint_result` as parameter ?

Reply:
not needed as this function is not called by other functions